server/bleep/src/webserver/answer.rs

```diff
@@ -44,6 +44,8 @@ use exchange::{Exchange, SearchStep, Update};
 use llm_gateway::api::FunctionCall;
 
 const TIMEOUT_SECS: u64 = 60;
+const ANSWER_MODEL: &str = "gpt-4";
+const ANSWER_MODEL_WITH_REV: &str = "gpt-4-0613";
```

Comment:
Why do we have both of these constants, instead of just always using `gpt-4-0613`?

Reply:
righto, looks like tiktoken looks for a prefix.